lib/surgex/repo_helpers/repo_helpers.ex

```diff
@@ -83,4 +84,18 @@ defmodule Surgex.RepoHelpers do
       _ -> opts
     end
   end
+
+  @doc """
+  Sets application_name to the value of APP_NAME env var.
+  """
+  def set_application_name(opts) do
+    with env_value when is_binary(env_value) <- System.get_env("APP_NAME"),
+         application_name <- String.slice(env_value, 0, 63),
```

Comment:
Why do we limit to 63 characters?

Reply:
Quoting postgres docs:
>The application_name can be any string of less than NAMEDATALEN characters (64 characters in a standard build). 

https://www.postgresql.org/docs/13/runtime-config-logging.html#GUC-APPLICATION-NAME